ThePhasesOfCraftship/2_best_practice_first/strategicDesignPart1/exercises/1_RefactoringTo4Tiers/begin/src/persistence/index.ts

```diff
@@ -0,0 +1,176 @@
+import { PrismaClient } from "@prisma/client";
+
+interface StudentPersistence {
```

Comment:
I find these methods returning `any` "too unsafe". I think by unsafe refactoring we mean refactoring without tests that ensure the functionality is unchanged. I think it would be better to keep the project at least as typesafe as it was before.